lib/html-composer.js

```diff
@@ -94,6 +96,12 @@ function applyOptions(options, res) {
 
     // can't assign to result, or we'd get a circular reference in refs.
     _.set(state, '_components', componentList);
+    _.set(state, '_componentSchemas', _.map(componentList, function (name) {
+      return {
+        name: name,
+        schema: schema.getSchema(files.getComponentPath(name))
+      };
+    }));
```

Comment:
Is kiln the only component interested in this data? I am wondering how this will affect page-load performance, as it increases the size of the state object. Perhaps there could be another end-point to return all the schemas which kiln could call.